rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1806,4 +1806,36 @@ private void unselectNodes(final HashSet<RMNode> selectedRMNodes) {
         }
     }
 
+    protected void setNodeSources(HashMap<String, NodeSource> hashMap) {
```

Comment:
Yeah I wanted to use PowerMock but I remembered we had problems with this library at some point